app/i18n/app.fr.i18n.json

```diff
@@ -1,7 +1,7 @@
 {
     "app": {
         "loading": "Chargement...",
-        "offline": "Impossible de se connecter, êtes-vous hors-ligne?",
+        "offline": "Impossible de se connecter, êtes-vous hors-ligne ?",
```

Comment:
Yes indeed !

It is the rule in french. More accurately it is a non breaking space &nbsp; to avoid line break :)

> Le 18 mai 2016 à 18:24, Alex Van de Sande <notifications@github.com> a écrit :
> 
> In app/i18n/app.fr.i18n.json <https://github.com/ethereum/meteor-dapp-wallet/pull/190#discussion_r63735878>:
> 
> > @@ -1,7 +1,7 @@
> >  {
> >      "app": {
> >          "loading": "Chargement...",
> > -        "offline": "Impossible de se connecter, êtes-vous hors-ligne?",
> > +        "offline": "Impossible de se connecter, êtes-vous hors-ligne ?",
> Personally I hate spaces before punctuation, but I do understand this is default in french typesetting..
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/ethereum/meteor-dapp-wallet/pull/190/files/9565b07b6ba1a347dc3334abe6052687fbd55ac5#r63735878>